{"uid":"327b35264bd68c1a","name":"test_database_get_by_name","fullName":"tests.unit.V1.service.test_database#test_database_get_by_name","historyId":"7dd5153d766952689de904957a11cd32","time":{"start":1730516816307,"stop":1730516816346,"duration":39},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","statusTrace":"self = <allure_pytest.listener.AllureListener object at 0x7fd72b108490>\nfixturedef = <FixtureDef argname='database_get_callback' scope='function' baseid='tests/unit/V1/service'>\n\n    @pytest.hookimpl(hookwrapper=True)\n    def pytest_fixture_post_finalizer(self, fixturedef):\n        yield\n        if hasattr(fixturedef, 'cached_result') and self._cache.get(fixturedef):\n            container_uuid = self._cache.pop(fixturedef)\n>           self.allure_logger.stop_group(container_uuid, stop=now())\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_pytest/listener.py:190: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/reporter.py:91: in stop_group\n    plugin_manager.hook.report_container(container=group)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:34: in report_container\n    self._report_item(container)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:25: in _report_item\n    with io.open(self._report_dir / filename, 'w', encoding='utf8') as json_file:\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_io.py:93: in open\n    return fake_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:105: in fake_open\n    return fake_file_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:136: in __call__\n    return self.call(*args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:244: in call\n    file_object = self._init_file_object(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:358: in _init_file_object\n    file_object = self.filesystem.create_file_internally(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:2551: in create_file_internally\n    self.raise_os_error(errno.ENOENT, parent_directory)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pyfakefs.fake_filesystem.FakeFilesystem object at 0x7fd728e760a0>\nerr_no = 2\nfilename = '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\nwinerror = None\n\n    def raise_os_error(\n        self,\n        err_no: int,\n        filename: Optional[AnyString] = None,\n        winerror: Optional[int] = None,\n    ) -> NoReturn:\n        \"\"\"Raises OSError.\n        The error message is constructed from the given error code and shall\n        start with the error string issued in the real system.\n        Note: this is not true under Windows if winerror is given - in this\n        case a localized message specific to winerror will be shown in the\n        real file system.\n    \n        Args:\n            err_no: A numeric error code from the C variable errno.\n            filename: The name of the affected file, if any.\n            winerror: Windows only - the specific Windows error code.\n        \"\"\"\n        message = os.strerror(err_no) + \" in the fake filesystem\"\n        if winerror is not None and sys.platform == \"win32\" and self.is_windows_fs:\n            raise OSError(err_no, message, filename, winerror)\n>       raise OSError(err_no, message, filename)\nE       FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:489: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"global_fake_fs","time":{"start":1730516816160,"stop":1730516816165,"duration":5},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_cache","time":{"start":1730516816160,"stop":1730516816160,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","statusTrace":"self = <allure_pytest.listener.AllureListener object at 0x7fd72b108490>\nfixturedef = <FixtureDef argname='database_get_callback' scope='function' baseid='tests/unit/V1/service'>\n\n    @pytest.hookimpl(hookwrapper=True)\n    def pytest_fixture_post_finalizer(self, fixturedef):\n        yield\n        if hasattr(fixturedef, 'cached_result') and self._cache.get(fixturedef):\n            container_uuid = self._cache.pop(fixturedef)\n>           self.allure_logger.stop_group(container_uuid, stop=now())\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_pytest/listener.py:190: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/reporter.py:91: in stop_group\n    plugin_manager.hook.report_container(container=group)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:34: in report_container\n    self._report_item(container)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:25: in _report_item\n    with io.open(self._report_dir / filename, 'w', encoding='utf8') as json_file:\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_io.py:93: in open\n    return fake_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:105: in fake_open\n    return fake_file_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:136: in __call__\n    return self.call(*args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:244: in call\n    file_object = self._init_file_object(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:358: in _init_file_object\n    file_object = self.filesystem.create_file_internally(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:2551: in create_file_internally\n    self.raise_os_error(errno.ENOENT, parent_directory)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pyfakefs.fake_filesystem.FakeFilesystem object at 0x7fd728e760a0>\nerr_no = 2\nfilename = '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\nwinerror = None\n\n    def raise_os_error(\n        self,\n        err_no: int,\n        filename: Optional[AnyString] = None,\n        winerror: Optional[int] = None,\n    ) -> NoReturn:\n        \"\"\"Raises OSError.\n        The error message is constructed from the given error code and shall\n        start with the error string issued in the real system.\n        Note: this is not true under Windows if winerror is given - in this\n        case a localized message specific to winerror will be shown in the\n        real file system.\n    \n        Args:\n            err_no: A numeric error code from the C variable errno.\n            filename: The name of the affected file, if any.\n            winerror: Windows only - the specific Windows error code.\n        \"\"\"\n        message = os.strerror(err_no) + \" in the fake filesystem\"\n        if winerror is not None and sys.platform == \"win32\" and self.is_windows_fs:\n            raise OSError(err_no, message, filename, winerror)\n>       raise OSError(err_no, message, filename)\nE       FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:489: FileNotFoundError","steps":[],"attachments":[{"uid":"9953399e8e955ae0","name":"log","source":"9953399e8e955ae0.txt","type":"text/plain","size":246}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"global_fake_fs::0","time":{"start":1730516816380,"stop":1730516816381,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests.unit.V1.service"},{"name":"suite","value":"test_database"},{"name":"host","value":"fv-az849-425"},{"name":"thread","value":"2089-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.unit.V1.service.test_database"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":49,"skipped":0,"passed":58,"unknown":0,"total":107},"items":[{"uid":"de5650037ff45183","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/342//#testresult/de5650037ff45183","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730431060850,"stop":1730431060888,"duration":38}},{"uid":"fe740447f22dd551","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/345//#testresult/fe740447f22dd551","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730430890493,"stop":1730430890530,"duration":37}},{"uid":"8efbfc7dbc08abfc","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/341//#testresult/8efbfc7dbc08abfc","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730344333772,"stop":1730344333812,"duration":40}},{"uid":"17451ff8fc052769","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/344//#testresult/17451ff8fc052769","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730344203070,"stop":1730344203109,"duration":39}},{"uid":"a2e1684113d2fca3","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/340//#testresult/a2e1684113d2fca3","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730257886998,"stop":1730257887035,"duration":37}},{"uid":"df3e029dd8aeb043","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/343//#testresult/df3e029dd8aeb043","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730257745870,"stop":1730257745909,"duration":39}},{"uid":"814fa328fe364a94","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/339//#testresult/814fa328fe364a94","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730171516165,"stop":1730171516217,"duration":52}},{"uid":"7fa9f02a75d9ebab","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/342//#testresult/7fa9f02a75d9ebab","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730171378269,"stop":1730171378308,"duration":39}},{"uid":"d5f2ef945827e626","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/338//#testresult/d5f2ef945827e626","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730085248614,"stop":1730085248652,"duration":38}},{"uid":"ad83355f76f31484","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/341//#testresult/ad83355f76f31484","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730085095031,"stop":1730085095069,"duration":38}},{"uid":"6537beecd44aa4c5","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/337//#testresult/6537beecd44aa4c5","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729998798021,"stop":1729998798057,"duration":36}},{"uid":"af47237744bf4ea7","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/340//#testresult/af47237744bf4ea7","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729998666823,"stop":1729998666862,"duration":39}},{"uid":"1c814c34e02730ad","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/336//#testresult/1c814c34e02730ad","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729912104959,"stop":1729912104997,"duration":38}},{"uid":"70075af2dcd19204","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/339//#testresult/70075af2dcd19204","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729911964678,"stop":1729911964717,"duration":39}},{"uid":"b601005e81078d0","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/335//#testresult/b601005e81078d0","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729825916138,"stop":1729825916177,"duration":39}},{"uid":"f5f27a7d7a21c0c7","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/338//#testresult/f5f27a7d7a21c0c7","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729825778510,"stop":1729825778549,"duration":39}},{"uid":"f926827ae22f83eb","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/334//#testresult/f926827ae22f83eb","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729739413336,"stop":1729739413375,"duration":39}},{"uid":"6eb67068450dbdc3","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/337//#testresult/6eb67068450dbdc3","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729739265208,"stop":1729739265244,"duration":36}},{"uid":"6c8b004cffb4e7b6","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/333//#testresult/6c8b004cffb4e7b6","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729653022538,"stop":1729653022576,"duration":38}},{"uid":"86ad72e241f8b780","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/336//#testresult/86ad72e241f8b780","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729652880315,"stop":1729652880355,"duration":40}}]},"tags":[]},"source":"327b35264bd68c1a.json","parameterValues":[]}